### PR TITLE
Feature/dashboard rating

### PR DESCRIPTION
**Pull request by mujtaba912**

### CODE REVIEW

**Review by devsadeq — APPROVED** [review]

File: client_dashboard/common/src/commonMain/kotlin/org/thechance/common/components/EditableRatingBar.kt [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+    rating: Double,
+    count: Int,
+    selectedIcon: Painter,
+    halfSelectedIcon: Painter,
+    notSelectedIcon: Painter,
+    iconsSize: Dp = 24.dp,
+    iconsPadding: Dp = 0.dp,
+    horizontalArrangement: Arrangement.Horizontal = Arrangement.Start,
+    onClick: (Double) -> Unit,
+    modifier: Modifier = Modifier,
```
Comment (ay0o5h) [inline client_dashboard/common/src/commonMain/kotlin/org/thechance/common/components/EditableRatingBar.kt:32]: add modifier as first optional parameter

File: client_dashboard/common/src/commonMain/kotlin/org/thechance/common/components/RatingBar.kt [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+@Composable
+fun RatingBar(
+    rating: Double,
+    selectedIcon: Painter,
+    halfSelectedIcon: Painter,
+    iconsSize: Dp = 24.dp,
+    iconsPadding: Dp = 0.dp,
+    horizontalArrangement: Arrangement.Horizontal = Arrangement.Start,
+    modifier: Modifier = Modifier,
```
Comment (ay0o5h) [inline client_dashboard/common/src/commonMain/kotlin/org/thechance/common/components/RatingBar.kt:25]: also add modifier as first optional parameter

**Review by ay0o5h — CHANGES_REQUESTED** [review]

File: client_dashboard/common/src/commonMain/kotlin/org/thechance/common/components/EditableRatingBar.kt (hunk 1 above)
Comment (mujtaba912) [reply client_dashboard/common/src/commonMain/kotlin/org/thechance/common/components/EditableRatingBar.kt]: done

File: client_dashboard/common/src/commonMain/kotlin/org/thechance/common/components/RatingBar.kt (hunk 2 above)
Comment (mujtaba912) [reply client_dashboard/common/src/commonMain/kotlin/org/thechance/common/components/RatingBar.kt]: done

**Review by ay0o5h — APPROVED** [review]
well done 👏🏻

**Review by aya-seif — APPROVED** [review]

**Pull request merged**